periodic_job_test.go

```diff
@@ -18,7 +18,7 @@ func TestNeverSchedule(t *testing.T) {
 		t.Parallel()
 
 		schedule := NeverSchedule()
-		require.Equal(t, time.Unix(1<<63-1, 0), schedule.Next(time.Now()))
+		require.Equal(t, time.Unix(1<<63-62135596801, 999999999), schedule.Next(time.Now()))
```

Comment:
Maybe we should add one more test here to verify that it’s more than 1000 years in the future or something like that?

Reply:
I'll add that plus check if the next schedule is  not before time.Now

Reply:
Also I think there might be the same issue with PeriodicInterval with a large duration. I'll take a look at this

Reply:
Done